src/Microsoft.AspNet.TestHost/ClientHandler.cs

```diff
@@ -23,14 +24,15 @@ namespace Microsoft.AspNet.TestHost
     /// </summary>
     public class ClientHandler : HttpMessageHandler
```

Comment:
The default http context factory is less convienet to use here, but it would make it so you don't have to think about the accessor.